site/docs/zh/plugins/i18n.md

```diff
@@ -395,6 +395,8 @@ interface SessionData {
 
 type MyContext = Context & SessionFlavor<SessionData> & I18nFlavor;
 
+const bot = new Bot<MyContext>(""); // <-- 把你的 bot token 放在这里
```

Comment:
Is this comment supposed to be here?

Reply:
Ask the Chinese. This comment was in the example for the tip script, so I decided to leave it here as well.

Reply:
It might be a mistake from earlier, I'll wait for @roj1512 to reply

Reply:
@Nazar-Ant Can you also open a PR to remove that?